cmd/protoc-gen-openapi/examples/tests/protobuftypes/openapi.yaml

```diff
@@ -246,7 +246,7 @@ components:
                 - type: array
                   items:
                     $ref: '#/components/schemas/AnyJSONValue'
```

Comment:
On my first read-through, I'm wondering why this didn't change to google.protobuf.Value

Reply:
This file was generated with `fq_schema_naming=false` (the default) so it uses the older `AnyJSONValue` schema name for backwards compatibility.

Reply:
There was a bug in my previous PR with `google.protobuf.Value` not getting converted to the fully qualified name, which I didn't catch until I added support for `google.protobuf.Any`. This is why you see the schema name change in the `openapi_fq_schema_naming.yaml` file (which uses fully qualified naming) since I have fixed the issue in this PR.